src/App.css

```
@@ -50,4 +52,17 @@ a {
 *:before,
 *:after {
   box-sizing: border-box;
+}
+
+button {
+  background: none;
+  border: 0;
+  font-size: 18px;
+  padding: 0;
+  text-transform: capitalize;
+  cursor: pointer;
+}
+
+button:hover {
+  text-decoration: underline;
 }
```

Not necessary. I like newlines at the end of files
```suggestion
}

```